tests/test_link_io_structure.py

```diff
@@ -95,7 +95,30 @@ def test_overwrite_symlink_stdout(tmpdir, capsys):
         os.symlink("2023.06", "2023")
         create_symlinks(links_dict)
         captured = capsys.readouterr()
-        assert "Existing symlink 2023 moved from 2023.06 to 2023.07\n" == captured.out
+        assert (
+            "Existing symlink 2023 moved from 2023.06 to 2023.07"
+            in captured.out.split("\n")
+        )
+
+
+def test_overwrite_symlink_implicit_multiple_stdout(tmpdir, capsys):
+    links_dict = {
+        "root_folder": tmpdir,
+        "links": {"a": "d", "d": "e", "b": "c", "c": "d", "e": "g"},
+    }
+    with tmpdir.as_cwd():
+        os.mkdir("f")
+        os.mkdir("g")
+        os.symlink("d", "a")
+        os.symlink("c", "b")
+        os.symlink("d", "c")
+        os.symlink("e", "d")
+        os.symlink("f", "e")
+        create_symlinks(links_dict)
+        captured = capsys.readouterr()
+        for symlink in list("ab"):
+            assert f"{symlink} was implicitly moved" in captured.out.split("\n")
```

Comment:
This example with single letters is fine, but it is hard to read and verify for correctness. Can you make another example that is using `stable` -> `stable-py38` -> `2023.09.03` f.ex and assert exactly on the full output?

Reply:
Yes, certainly. It was easier to read when I had the drawing next to me on my desk